source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/test/lambda-elasticsearch-kibana.test.ts

```diff
@@ -146,4 +131,333 @@ test('check override cognito domain name with provided cognito domain name', ()
   expect(stack).toHaveResource('AWS::Cognito::UserPoolDomain', {
     Domain: 'test-cognito-domain'
   });
-});
\ No newline at end of file
+});
+
+test('Check for 3 subnets for environment specified stack', () => {
+  const stack = new cdk.Stack(undefined, undefined, {
+    env: { account: "123456789012", region: 'us-east-1' },
+  });
+
+  const lambdaProps: lambda.FunctionProps = {
```

Comment:
hmm - why not use GetDefaultTestLambdaProps here? Am I missing a subtle difference? The upside is that we use Node 14 instead of 12, which is usually good.